openpathsampling/random.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import random
```

Comment:
added the random import here to (hopefully) make the import less confusing for `python2.7`